fendermint/vm/ipc_actors/build.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright 2021-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use std::io::Write;
+use std::path::{Path, PathBuf};
+
+/// Generate Rust bindings from the IPC Solidity Actors ABI artifacts.
+///
+/// These are built by `make ipc-actors-abi`, here we just add the final step
+/// so we have better code completion with Rust Analyzer.
+fn main() {
```

Comment:
@cryptoAtwill, actually, what about removing the type bindings from the agent and leveraging the ones in Fendermint (the same way that we leverage the ones from `ipc-actors` to interact with the native actors?